command/helpers.go

```diff
@@ -53,6 +53,10 @@ func limit(s string, length int) string {
 
 // formatTime formats the time to string based on RFC822
 func formatTime(t time.Time) string {
+	if t.Unix() < 1 {
```

Comment:
I did that at first and it didn't work because `time.Time{} != time.Unix(0,0)` and we default to the latter. `t.Unix() < 1` catches both.